connect_ext_test.go

```diff
@@ -1510,7 +1510,8 @@ func TestBidiStreamServerSendsFirstMessage(t *testing.T) {
 
 func TestStreamForServer(t *testing.T) {
 	t.Parallel()
-	newPingClient := func(pingServer pingv1connect.PingServiceHandler) pingv1connect.PingServiceClient {
+	newPingClient := func(t *testing.T, pingServer pingv1connect.PingServiceHandler) pingv1connect.PingServiceClient {
+		t.Helper()
```

Comment:
I think you can undo all of the changes in this file, right? Looks like the addition of `t` here is unused, and this appears to be the only change in here.

Reply:
The servers close are matched to the clients which is nice. The other change in this file is the two `Receives` for the test cases that `Send` a non nil message.

Reply:
As in `server := memhttptest.NewServer(t, mux)` takes the correct subtests `*testing.T` rather than the parent.

Reply:
Ah, okay. That was a bit subtle due to the shadowing of the name `t`.